mod/friends_collections/views/default/collections/collection.php

```diff
@@ -57,7 +57,7 @@
 
 $params = [
 	'collection' => $collection,
-	'metadata' => $menu,
+	'metadata' => $full_view ? false : $menu,
```

Comment:
This is not the correct place to fix this. 
As i requested in my previous comment please do this around line 41. It'll be more stable in the future because it's easier to read what happens.